### PR TITLE
chore: update helm chart to include driver:v0.0.12

### LINKED ISSUE #134: [helm installation] [Create k8s service account for the azure-provider daemonset]
**Describe the solution you'd like**
right now the azure provider daemonset is assigned to `default` service-account in k8s namespace which makes it hard for us to create separate pod security policy only for azure provider daemonset.

in the helm templates, could you please add/create k8s service account and assign it to the azure provider daemonset. 

**Anything else you would like to add:**
here's the slack thread https://kubernetes.slack.com/archives/C013PUP2WRK/p1592907629020400

**Environment:**

- Secrets Store CSI Driver version: 0.0.11
- Azure Key Vault provider version: 0.0.6
- Kubernetes version: 1.16.9
- Cluster type: AKS

### PR DESCRIPTION
**Pull request by aramase**
**What this PR does / why we need it**:
- Updates helm chart dep for driver to `v0.0.12`
- Adds a new service account to use with the provider installer pods

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #134 

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/csi-secrets-store-provider-azure/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{ if .Values.rbac.install }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-secrets-store-provider-azure
```
Comment (ritazh) [inline charts/csi-secrets-store-provider-azure/templates/serviceaccount.yaml:5]: Where do we use this serviceAccount in the chart deployment yamls?

Comment (aramase) [reply charts/csi-secrets-store-provider-azure/templates/serviceaccount.yaml:5]: Had added it in `deployment/*` dir, missed out the charts. Updated!

**Pull request merged**